hw/ip/otbn/doc/_index.md

```diff
@@ -110,6 +110,7 @@ Providing the stack has at least one element, this is allowed, even if the stack
 Control and Status Registers (CSRs) are 32b wide registers used for "special" purposes, as detailed in their description;
 they are not related to the GPRs.
 CSRs can be accessed through dedicated instructions, `CSRRS` and `CSRRW`.
+Writes to read-only registers are ignored, they do not signal an error.
```

Comment:
Nit: this should be a semicolon or a full stop (and the same for the other hunk).